tests/README.rst

```diff
@@ -1,10 +1,18 @@
 Testing Anaconda
 ================
 
-Testing locally
----------------
+This document describes how to run Anaconda tests. Anaconda has various tests for example
+unit tests, rpm tests, translation tests. All the tests will be run together if you follow
+steps below.
 
-Before testing Anaconda you need to install all required dependencies.
+You have two alternate ways how to run these tests. First is directly on your system or
```

Comment:
I suggest using bullet points for the two way:

- running the test directly on your system
- using mock utility which run the tests in a container on your system

Read bellow about benefits and drawbacks of the two methods.

Reply:
Nice improvement. Thanks!